src/OpenLoco/src/GameSaveCompare.cpp

```diff
@@ -124,6 +124,32 @@ namespace OpenLoco::GameSaveCompare
         return divergentBytesTotal > 0;
     }
 
+    static bool isLoggedDivergenceUserString(const std::string type, const std::span<char[32]> lhsArr, const std::span<char[32]> rhsArr, bool displayAllDivergences)
```

Comment:
I've added this as we clear the user string when adding a new one CS will just copy only the required bytes for the new string leaving any remaining bytes from the old string.